tests/output/cs/60015-UNI-14131.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+
+class C
+{
+    public void S()
+    {
+        Action localMethod = () => {
+                SomeClass.OtherMethod(new Dictionary<string, string>
+            {
+                {"a", "one"},
+                {"b", "two"},
+                {"c", "three"}
+            });
+            };
```

Comment:
@mihaipopescu is this really the output that should be generated? Looks strange to me ...